config/locales/defaults/responders/en.yml

```diff
@@ -34,6 +34,10 @@ en:
     students:
       create:
         error: "%{resource_name} could not be created due to the following error(s): %{errors}."
+      destroy:
+        error: 'Error deleting Student (%{user_name}): %{message}'
```

Comment:
Two comments overall:

1. Don't capitalize "Student"
2. Let's change the verb "delete" to "remove". I think that has a more accurate connotation. You can make the wording change for TAs as well.